src/NuGet.Core/NuGet.ProjectModel/CacheFileProperties.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.ProjectModel
+{
+    internal class CacheFileProperties
+    {
+        internal const string VersionProperty = "version";
```

Comment:
The only place these properties are used are in `CacheFile`, in `JsonPropertyName` attributes. I think we're better off just putting the strings directly in the attribute (for example `[JsonPropertyName("version")]`), rather than having this class.